SR2019/2018-10-10-place-confirmation.md

```diff
@@ -0,0 +1,11 @@
+---
+to: Student Robotics 2019
+subject: SR2019 Registration and Kickstart Announcement
+note: This email will be sent multiple times, to those in the list not in the "Contacted" segment
+---
+
+You're in! This email is a confirmation of your teams place at Student Robotics 2019. We are unable to confirm second teams until our deadline of the 26th, due to the space constraints of the venue.
```

Comment:
"This email is a confirmation...." is a bit of a weird way to put it, "We're happy to confirm you've got a place at Student Robotics 2019!" would read better

Reply:
Also, "If you've applied for a second team, we sadly can't confirm you will receive a second kit until our deadline on the 26th"